src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -794,10 +793,9 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                     // it. However, it's deliberate.  We want to make sure that the code runs on 
                     // the background so that no one takes an accidentally dependency on running on 
                     // the UI thread.
-                    var refactorings = Task.Run(
+                    var refactorings = this.ThreadingContext.JoinableTaskFactory.Run(
                         () => _owner._codeRefactoringService.GetRefactoringsAsync(
```

Comment:
❗ Missing a switch to the background thread.

Reply:
done.